src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -266,7 +266,7 @@ class LicenceService(
     )
     licenceRepository.saveAndFlush(licenceEntity)
 
-    // if previous status was APPROVED and the new status is IN_PROGRESS then email OMU regarding re-approval
+    // if previous status was APPROVED and the new status is SUBMITTED then email OMU regarding re-approval
     if (isReApproval) {
       notifyReApprovalNeeded(licenceEntity)
```

Comment:
isReApproval a good term? Is there a better word?

Reply:
`isReApproval` => `isReApprovalRequired` ?